samples/Store/Domain/Cart.fs

```diff
@@ -131,11 +131,14 @@ type Accumulator<'event, 'state>(fold: 'state -> 'event seq -> 'state, originSta
         return result }
 #else
 let interpretMany fold interpreters (state: 'state): 'state * 'event list =
-    ((state,[]),interpreters)
-    ||> Seq.fold (fun (state: 'state, acc: 'event list) interpret ->
+    let mutable result = []
+    let mutable state = state
+    interpreters
+    |> Seq.iter (fun interpret ->
         let events = interpret state
-        let state' = fold state events
-        state', acc @ events)
+        result <- result @ events
+        state <- fold state (Seq.toArray events))
+    state, List.ofSeq result
```

Comment:
should accumulate into a ResizeArray (esp given result is actually a list here!)